src/core/expression/qgsexpression.cpp

```diff
@@ -830,6 +830,10 @@ void QgsExpression::initVariableHelp()
   sVariableHelpTexts.insert( QStringLiteral( "symbol_color" ), QCoreApplication::translate( "symbol_color", "Color of symbol used to render the feature." ) );
   sVariableHelpTexts.insert( QStringLiteral( "symbol_angle" ), QCoreApplication::translate( "symbol_angle", "Angle of symbol used to render the feature (valid for marker symbols only)." ) );
 
+  sVariableHelpTexts.insert( QStringLiteral( "symbol_label" ), QCoreApplication::translate( "variable_help", "Label of the symbol, user defined label will be used, otherwise will default to the label made by QGIS." ) );
+  sVariableHelpTexts.insert( QStringLiteral( "symbol_id" ), QCoreApplication::translate( "variable_help", "Id of the symbol." ) );
+  sVariableHelpTexts.insert( QStringLiteral( "symbol_count" ), QCoreApplication::translate( "variable_help", "Total number of features defined by this symbol." ) );
```

Comment:
```suggestion
  sVariableHelpTexts.insert( QStringLiteral( "symbol_count" ), QCoreApplication::translate( "symbol_count", "Total number of features defined by this symbol." ) );
```

Reply:
Thanks, I was not sure what to provide for that element give than most other were help and only a few had unique string,